src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2383,18 +2417,14 @@ public ConstructorEdit(INamedTypeSymbol oldType)
                                     // or is replacing an implicitly generated one (e.g. parameterless constructor, auto-generated record methods, etc.)
                                     oldContainingType = oldSymbol.ContainingType;
 
-                                    if (oldSymbol.IsImplicitlyDeclared)
+                                    if (oldSymbol.IsImplicitlyDeclared && oldSymbol.DeclaredAccessibility != newSymbol.DeclaredAccessibility)
```

Comment:
> && oldSymbol.DeclaredAccessibility != newSymbol.DeclaredAccessibility [](start = 70, length = 70)

Why this change? The previous code meant to exclude implicitly declared symbols from all the cases below.

Reply:
Turns out I was only half right with what I said earlier today. `oldSymbol.IsImplicitlyDeclared` is true when explicitly implementing methods from records, like PrintMembers(), ToString() etc., and whilst it isn't true for properties themselves, it is true for get and set accessors of properties.

Reply:
ok

Reply:
Updated this so that implicitly declared cases are handled here instead of below.